propagation/intercomparison.md

```diff
@@ -6,3 +6,24 @@ to choose.
 
 Additionally, it assists in identifying the most suitable algorithms for 'cloudification' or enhancement, ensuring that
 only the most effective and efficient options are pursued.
+
+The APEx framework allows to simplify intercomparison, by offering different implementations of the same algorithm through 
+the same interface. One such example is Sentinel-1 backscatter computation, of which we already have 3 implementations available, 
+that are all exposed via the `sar_backscatter' openEO process:
+
+- Sentinel-1 toolbox 'SNAP' implementation on EODC's openEO backend.
+- Orfeo implementation on the CDSE openEO backend
+- A custom implementation on the Sentinelhub, exposed via the Terrascope openEO backend.
+
+All three implementations implement sar_backscatter, but may support different parameters. Hence the range of supported
+parameters is an important point of comparision, but also the cost in platform credits as well as correctness of the results
+can be easily compared.
+
+## Call for service providers
+
+We are looking for a first category of algorithms to be intercompared. If you have an algorithm, consider onboarding it
+via the [Toolbox cloudification](./toolboxcloud.md) service, and request your peers to do the same so a first 
```

Comment:
This probably has to be reviewed based on discussion going on in https://github.com/ESA-APEx/apex_documentation/pull/17

Reply:
yes, but not a blocker to merge this one already?